style.css

```diff
@@ -27,7 +27,19 @@ aside {
 }
 
 .puzzle-section {
+  display: flex;
   height: 400px;
+  justify-content: center;
+}
+
+.puzzle-section p {
+  border: 2px solid tomato;
+  font-size: 27px;
+  height: 50px;
+  padding-top: 7px;
+  text-transform: capitalize;
+  text-align: center;
+  width: 50px;
```

Comment:
If you haven't worked with `rems` and `ems` already, I would encourage you to do so!